cob_hardware_config/cob4-2/config/teleop.yaml

```diff
@@ -72,7 +72,7 @@ components: {
   head: {
     sss_default_target: "front",
     velocity_topic_name: "/head/joint_group_velocity_controller/command",
-    joint_velocity: [0.05, 0.05],
+    joint_velocity: [0,05, 0.05, 0.05],
```

Comment:
@ipa-nhg 
why is head included in teleop.yaml for a robot that does not have a head?
can the tag be removed? what about all the parameters that have "head" in its name (e.g. `head_sss_default_target_button`)?